config/sync/field.field.paragraph_row.node_stanford_page_row.su_page_components.yml

```diff
@@ -40,18 +40,27 @@ settings:
         weight: -12
         enabled: false
       stanford_media_caption:
-        weight: -16
+        weight: 8
         enabled: false
       stanford_person_cta:
         enabled: true
-        weight: -15
+        weight: 11
       stanford_schedule:
         enabled: true
-        weight: -13
+        weight: 12
```

Comment:
This is another thing where I feel like maybe we should just make them match for simplicity, unless there's a strong reason not to.

Reply:
I guess that's precisely what's happening here.  Thumbs up on that.